CHANGELOG.md

```diff
@@ -5,143 +5,152 @@ All notable changes to this project will be documented in this file.
+- SID change triggered MAC address generation [`#20`](https://github.com/3mdeb/meta-rte/pull/20)
 
 ## [v0.7.0](https://github.com/3mdeb/meta-rte/compare/v0.6.2...v0.7.0) - 2021-07-29
 
 ### Merged
 
-- Release v0.7.0 [#19](https://github.com/3mdeb/meta-rte/pull/19)
-- Issue templates [#18](https://github.com/3mdeb/meta-rte/pull/18)
-- RTE features setup [#10](https://github.com/3mdeb/meta-rte/pull/10)
-- MAC address recalculation [#17](https://github.com/3mdeb/meta-rte/pull/17)
-- docs/development.md: fix minor spelling and broken link [#13](https://github.com/3mdeb/meta-rte/pull/13)
-- packagegroup-rte.bb: add xradio, xradio-firmware and wpa_supplicant [#8](https://github.com/3mdeb/meta-rte/pull/8)
-- U boot 2020.04 [#7](https://github.com/3mdeb/meta-rte/pull/7)
-- Features veryfication: [#9](https://github.com/3mdeb/meta-rte/pull/9)
-- README: serial connection and telnet [#12](https://github.com/3mdeb/meta-rte/pull/12)
-- README: Add instructions for serial access [#11](https://github.com/3mdeb/meta-rte/pull/11)
-- wakeonlan [#6](https://github.com/3mdeb/meta-rte/pull/6)
-- openocd: install necessary config files [#5](https://github.com/3mdeb/meta-rte/pull/5)
-- Add openvpn [#4](https://github.com/3mdeb/meta-rte/pull/4)
-- fix links [#3](https://github.com/3mdeb/meta-rte/pull/3)
+- Release v0.7.0 [`#19`](https://github.com/3mdeb/meta-rte/pull/19)
+- Issue templates [`#18`](https://github.com/3mdeb/meta-rte/pull/18)
+- RTE features setup [`#10`](https://github.com/3mdeb/meta-rte/pull/10)
+- MAC address recalculation [`#17`](https://github.com/3mdeb/meta-rte/pull/17)
+- docs/development.md: fix minor spelling and broken link [`#13`](https://github.com/3mdeb/meta-rte/pull/13)
+- README.md: add information about serial connection and telnet [`#12`](https://github.com/3mdeb/meta-rte/pull/12)
+- README: Add instructions for serial access [`#11`](https://github.com/3mdeb/meta-rte/pull/11)
+- Wip wakeonlan [`#6`](https://github.com/3mdeb/meta-rte/pull/6)
+- recipes-devtools/openocd: install necessary config files [`#5`](https://github.com/3mdeb/meta-rte/pull/5)
+- Add openvpn [`#4`](https://github.com/3mdeb/meta-rte/pull/4)
+- Add documentation for meta-rte [`#56`](https://github.com/3mdeb/meta-rte/pull/56)
+- README.md: fix links [`#3`](https://github.com/3mdeb/meta-rte/pull/3)
 
 ## [v0.6.2](https://github.com/3mdeb/meta-rte/compare/v0.6.1...v0.6.2) - 2019-02-05
 
 ### Merged
 
 - release v0.6.2 [`#2`](https://github.com/3mdeb/meta-rte/pull/2)
 - GitHub release [`#1`](https://github.com/3mdeb/meta-rte/pull/1)
-- Add  swu image to automatic release [`#54`](https://github.com/3mdeb/meta-rte/compare/bcaa2fb646ce62323ae04055e3171caf48ac8c98%5E...f91624c07cdc66539225350f82ab283094115e5a)
-- release v0.6.1 [`#53`](https://github.com/3mdeb/meta-rte/commit/5f2cd76a265bd7ef8bafabc582b6a98b723062d0)
+- Add  swu image to automatic release [`#54`](https://github.com/3mdeb/meta-rte/pull/54)
```

Comment:
I'm wondering why there's quite a lot of changes in the old entries.

Reply:
@Silentsky0 How did you generate CHANGELOG?  Those changes now leads to pull requests that unfortunately does not exists because we needed to move repo from gitlab in the past.

Reply:
Maybe the generation method is fine, ut we should not incliude all changes in the commit (only include the current release change with `git add -p`.

Reply:
That's a good idea. @Silentsky0 let's add that information i.e. here https://github.com/3mdeb/meta-rte#individual-mr